### DISCUSSION TITLE
Dialog is cropped on smaller devices

### DISCUSSION THREAD

**Discussion by AdisonCavani** [mantinedev/mantine#2262] [Feedback]
``<Dialog/>`` component is cropped on smaller devices. There is no option to center it's position or make it smaller depending on device viewport.

See - [mantine.dev](https://mantine.dev) documentation example:

![302003850_595613621972185_6225966015342631710_n](https://user-images.githubusercontent.com/76855180/187080092-740ee99b-41f0-4b0f-9edd-522a7b913237.jpg)

**Comment by rtivital** [comment]
Thanks for reporting, the issue will be fixed in one of the next releases.
